README.md

```diff
@@ -27,19 +27,26 @@ npm install --save-dev eslint eslint-plugin-n
         "ecmaVersion": 2020
     },
     "rules": {
-        "n/exports-style": ["error", "module.exports"],
-        "n/file-extension-in-import": ["error", "always"],
-        "n/prefer-global/buffer": ["error", "always"],
-        "n/prefer-global/console": ["error", "always"],
-        "n/prefer-global/process": ["error", "always"],
-        "n/prefer-global/url-search-params": ["error", "always"],
-        "n/prefer-global/url": ["error", "always"],
-        "n/prefer-promises/dns": "error",
-        "n/prefer-promises/fs": "error"
+        "n/exports-style": ["error", "module.exports"]
     }
 }
 ```
 
+### [`eslint.config.js`](https://eslint.org/docs/latest/use/configure/configuration-files-new) (requires eslint>=v8.23.0)
+
+```js
+const nodeRecommendedScript = require("eslint-plugin-n/configs/recommended-script")
+
+module.exports = [
+    nodeRecommendedScript,
+    {
+        rules: {
+            "n/exports-style": ["error", "module.exports"]
+        }
+    }
+]
+```
```

Comment:
Maybe there should also be an example with `configs/recommended-module`, for ESM (`"type": "module"`) projects?